### PR TITLE
Feature/update styling

### PR DESCRIPTION
**Pull request by mbenfowler**
## What I did:
CSS updates!
Made changes mainly to fonts, Recommended component styling, and Login

On full screen, does anyone else see the Search component off to the right slightly?  I can't figure out why it's happening.

## Testing Notes/Extras: 
Test ALL user flows and make sure everything is still cohesive.

Did this break anything?

- [ ]  No
- [x]  Yes

Type of change

- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [x]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [x]  If this code needs to be tested, all tests are passing
- [ ]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [ ]  The code is DRY. If it's not, I tried my best
- [ ]  I reviewed my code before pushing

What's next?

### CODE REVIEW

**Review by alivaditis — APPROVED** [review]

File: src/components/Recommended/Recommended.css [hunk 1]
```diff
@@ -1,22 +1,43 @@
...
+.recommendedContainer h2 {
+    margin: 1em;
 }
 
 .album {
     display: flex;
     flex-direction: column;
     align-items: center;
+    margin-bottom: 10%;
+    font-size: x-large;
```
Comment (jalbe0076) [inline src/components/Recommended/Recommended.css:22]: This is messing up your container width on smaller screens. Remove the font size completely, I think it's too large on smaller screens anyways. Then move it into the media query and add it to `.album p` This should fix the issue of different size containers.

File: src/components/Nav/Nav.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ const Nav = () => {
         <NavLink to="/collections" className='nav'>COLLECTIONS</NavLink>
         <NavLink to="/journal" className='nav'>JOURNAL</NavLink>
         <NavLink to="/discover" className='nav'>DISCOVER</NavLink>
+        <p className='user-profile'>User</p>
```
Comment (jalbe0076) [inline src/components/Nav/Nav.js:12]: Can we move this `<p>` tag back out, I've completely changed this in my branch for the login and logout functionality.

**Review by jalbe0076 — CHANGES_REQUESTED** [review]
Great job! Just a few minor tweaks to deal with the different container size. I found the issue on my end. It's the text that's causing it. Solution in comments. 
Also, if someone keyborad bashes the journal comments run outside of the container. I think we should have the text wrap. I forget what the command is for this
image

File: src/components/Recommended/Recommended.css (hunk 1 above)
Comment (mbenfowler) [reply src/components/Recommended/Recommended.css:22]: I'm not seeing any issues on container width on smaller screens on my end - what exactly are you seeing?

File: src/components/Nav/Nav.js (hunk 2 above)
Comment (mbenfowler) [reply src/components/Nav/Nav.js:12]: Yah, the only reason I moved it in was it was impossible to center the tabs on the page - I'll revert and continue eagerly anticipating your changes 😬

File: src/components/Recommended/Recommended.css (hunk 1 above)
Comment (mbenfowler) [reply src/components/Recommended/Recommended.css:22]: oh you mean

**Comment by mbenfowler** [comment]
@jalbe0076 fixed up the Journal notes to allow wordwrapping long words too - Ready for another look!

**Pull request merged**